src/ui/analyze-popup.vue

```diff
@@ -0,0 +1,143 @@
+            @click="closePopup"
+          >
+            OK
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { veil } from './veil.js'
+import PopupHeader from './popup-header.vue'
+import { mapState } from 'vuex'
+import moment from 'moment'
+
+export default {
+  components: {
+    PopupHeader
+  },
+  props: {},
+  data: function () {
+    return {
+      popupTitle: 'Analyse'
+    }
+  },
+  computed: {
+    starttimeFormatted: function () {
+      return moment(this.timestamp.start).format('DD/MM/YYYY HH:mm')
+    },
+    endtimeFormatted: function () {
+      return moment(this.timestamp.end).format('DD/MM/YYYY HH:mm')
+    },
+    ...mapState([
+      'timestamp'
+    ])
+  },
+  mounted () {
+    const popupEle = document.querySelector('.analyze_popup_div')
+    if (popupEle) {
+      const disablePopupFunc = () => {
+        this.$store.commit('togglePopup', 'analyze')
+        window.MetricQWebView.instances[0].reload()
+      }
+      veil.create(disablePopupFunc)
+      veil.attachPopup(popupEle)
+    }
+    const table = document.getElementById('analyze_table')
+    this.createTable(table)
+    this.fillTable(table)
+  },
+  methods: {
+    closePopup (evt) {
+      veil.destroy(evt)
+    },
+    closePopupModal: function (evt) {
+      if (evt.target.getAttribute('role') === 'dialog') {
+        veil.destroy(evt)
+      }
+    },
+    createTable (table) {
+      table.innerHTML =
+          `<tr>
+            <th>Metrikname</th>
+            <th>Beschreibung</th>
+            <th>Min</th>
+            <th>Max</th>
+            <th>Avg</th>
+            <th>Einheit</th>
+          </tr>`
+    },
+    fillTable (table) {
+      for (const metricKey of this.$store.getters['metrics/getAllKeys']()) {
+        const metric = this.$store.getters['metrics/get'](metricKey)
+        const name = metric.htmlName
+        const desc = metric.description
+        const unit = metric.unit
+        const instance = window.MetricQWebView.instances[0]
+        instance.handler.metricQHistory.analyze(this.timestamp.start, this.timestamp.end).target(metricKey).run().then(data => {
+          const min = Object.values(data)[0].minimum.toLocaleString('de', {
+            minimumFractionDigits: 3,
+            maximumFractionDigits: 3
+          })
+          const max = Object.values(data)[0].maximum.toLocaleString('de', {
+            minimumFractionDigits: 3,
+            maximumFractionDigits: 3
+          })
+          const avg = Object.values(data)[0].mean.toLocaleString('de', {
+            minimumFractionDigits: 3,
+            maximumFractionDigits: 3
+          })
+          table.innerHTML += `<tr>
+            <td class="tdText">${name}</td>
+            <td class="tdText">${desc}</td>
+            <td class="tdNumber">${min}</td>
+            <td class="tdNumber">${max}</td>
+            <td class="tdNumber">${avg}</td>
+            <td>${unit}</td>
+        </tr>`
```

Comment:
Do we really need to build the html in JS here? I'd rather just extract the information and leave the html to the template.